spaces/domains.json

```
@@ -163,4 +163,5 @@
   "vote.resdx.io": "resdx.eth",
   "snapshot.solos.so": "solos.eth",
   "dao.pepemon.world": "pepedontdump.eth"
+  "vote.viralata.finance": "viralata.eth"
```

A comma is missing at the end of the line above your sub-domain.
```suggestion
  "dao.pepemon.world": "pepedontdump.eth",
  "vote.viralata.finance": "viralata.eth"
```